PatientRegistrationApp/src/components/HouseholdContactInfo.vue

```diff
@@ -1,12 +1,26 @@
 <template>
 	<v-container fluid>
+		<v-row>
+		<v-radio-group
+        required
+        :rules="[v => !!v || 'This field is required']"
+        v-model="phoneNumberAvailable"
+		>
+        <v-col align="right" cols="12">
+          <v-radio label="I have a phone number" value="yes" @change="PhoneNumberAvailable()"></v-radio>
+          <v-radio label="I do not have a phone number" value="no" @change="PhoneNumberNotAvailable()"></v-radio>
+		</v-col>
+		</v-radio-group>
+		</v-row>
+		
 		<v-row>
 			<v-col cols="12" sm="6" md="3">
 				<v-text-field
 					required
 					:rules="[v => !!v || 'Phone number is required']"
 					label="Primary Phone Number"
 					v-model="primaryPhoneNumber"
+					v-bind:disabled="phoneNumberDisabled"
```

Comment:
Looks good, but we need some more distinct way of shading or hiding the disabled fields. I couldn't tell they were disabled at first and they definitely don't pass the accessibility snuff test. I've seen google do some form of strike through on text (diagonal lines running through it all like your shading it). Maybe something like that? Really just looking for more apparent disabling of those fields... 

![image](https://user-images.githubusercontent.com/67079030/99348235-bbef6f80-285e-11eb-8ebd-922ef358ed58.png)